### PR TITLE
Page events

### PR DESCRIPTION
**Pull request by frankdiox**

### CODE REVIEW

File: src/documents_en/guide/overview.html [hunk 1]
```diff
@@ -1929,14 +1929,24 @@ <h4 id="Attributeevents">Event handler attributes</h4>
...
-`pageinit` event is called after [`<ons-page>`](/reference/ons-page.html) is initialized. Use this event to alter the behavior on each pages.
+[`<ons-page>`](/reference/ons-page.html) provides a set of DOM events that will be fired in different moments of its life cycle. Use these events to alter the behavior on each page.
+
+* `init` event is called after `<ons-page>` is attached to DOM. In order to provide backward compatibility, the old `pageinit` event is still fired but will be deprecated in upcoming versions.
+
+* `destroy` event is called before `<ons-page>` is destroyed and prior to DOM detachment.
+
+* `show` event is called after `<ons-page>` comes into view by changing its CSS, i.e. the page already exists in the page stack and is now shown. It is usually fired, for example, when popping pages with `<ons-navigator>` or swapping persistent tabs with `<ons-tabbar>`.
+
+* `hide` event is called after `<ons-page>` disappear from view by changing its CSS, i.e. the page is hidden but still exists in the page stack. It is usually fired, for example, when pushing pages with `<ons-navigator>` or swapping persistent tabs with `<ons-tabbar>`.
```
Comment (argelius) [inline src/documents_en/guide/overview.html]: disappears

**Comment by argelius** [comment]
@frankdiox Are these docs finished?

**Comment by frankdiox** [comment]
@argelius I think so. Do you want to add anything else?

**Comment by argelius** [comment]
I will merge them.

**Pull request merged**